frameworks/g-cloud-9/questions/services/serviceCategoriesHosting.yml

```diff
@@ -1,15 +1,34 @@
+depends:
+- being: [cloud-hosting]
+  'on': lot
+hint: You can choose multiple categories.
 id: serviceCategories
 name: Service categories
-question: Which categories does your service fit under?
-hint: You can choose multiple categories.
-
-depends:
-  - "on": lot
-    being:
-      - cloud-hosting
-
-type: checkboxes
+number_of_items: 20
```

Comment:
Hey, I think you've flipped the limits here.

![image](https://cloud.githubusercontent.com/assets/2454380/23413342/48ade856-fdd0-11e6-81ea-183b47d09f15.png)

I'm gonna leave the same comment on the other one for completeness, but it's basically just one comment.